package.json

```diff
@@ -13,7 +13,7 @@
     "test": "jest",
     "develop": "jest --watch"
   },
-  "testCaseJson": "https://raw.githubusercontent.com/w3c/wcag-act-rules/master/content-assets/wcag-act-rules/testcases.json",
+  "testCaseJson": "https://raw.githubusercontent.com/w3c/wcag-act-rules/main/content-assets/wcag-act-rules/testcases.json",
```

Comment:
I'm quite surprised this still works. The `master` branch was deleted several months ago. GH seems to just know to pull from main.